milvus/HttpClient.ts

```diff
@@ -76,22 +67,101 @@ export class HttpBaseClient {
 
   // database
   get database() {
-    return this.config.database || DEFAULT_DB;
+    return this.config.database ?? DEFAULT_DB;
   }
 
   // timeout
   get timeout() {
-    return this.config.timeout || DEFAULT_HTTP_TIMEOUT;
+    return this.config.timeout ?? DEFAULT_HTTP_TIMEOUT;
+  }
+
+  // headers
+  get headers() {
+    return {
+      Authorization: this.authorization,
+      Accept: 'application/json',
+      ContentType: 'application/json',
+    };
   }
 
-  get POST() {
-    return this.client.post;
+  // fetch
+  get fetch() {
+    return this.config.fetch ?? fetch;
   }
 
-  get GET() {
-    return this.client.get;
+  // POST API
+  async POST<T>(
+    url: string,
+    data: Record<string, any> = {},
+    options?: FetchOptions
+  ): Promise<T> {
+    try {
+      // timeout controller
+      const timeout = options?.timeout ?? this.timeout;
+      const abortController = options?.abortController ?? new AbortController();
+      const id = setTimeout(() => abortController.abort(), timeout);
+
+      // assign database
+      if (data) {
+        data.dbName = data.dbName ?? this.database;
+      }
+
+      const response = await this.fetch(`${this.baseURL}${url}`, {
+        method: 'post',
+        headers: this.headers,
+        body: JSON.stringify(data),
+        signal: abortController.signal,
+      });
+
+      clearTimeout(id);
+      return response.json() as T;
```

Comment:
Not sure, is all fetch client need res.json()?

Reply:
According to the spec, it should have the `json` method. 
https://developer.mozilla.org/en-US/docs/Web/API/Response